supervisor/daemon/daemon.go

```diff
@@ -75,7 +76,6 @@ func (d *Daemon) dialog(conn io.ReadWriter) {
 			answer.ok(metadata.VersionAsString())
 		case commandBye:
 			answer.ok("bye")
-			return
```

Comment:
I think this was left here for a reason? Did you remove it by accident or is there something else Im not seeing?

Reply:
This was on purpose. I don't see the purpose of this return statement.